nilearn/region_extraction/regionextractor.py

```diff
@@ -0,0 +1,232 @@
+"""
```

Comment:
We'll see naming later but `region_extraction/regionextractor.py` is a bit redundant. Also, we use `snake_case` as naming convention. So the name would be `region_extractor.py`.